icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -159,40 +208,53 @@ cd $ICU4C_DIR/source
 make all 2>&1 | tee /tmp/icu4c-oldData-makeAll.txt
 make check 2>&1 | tee /tmp/icu4c-oldData-makeCheck.txt
 
-# 4. Build the new ICU4C data files; these include .txt files and .py files.
-# These new files will replace whatever was already present in the ICU4C sources.
-# This process uses ant with ICU's data/build.xml and data/icu-config.xml to
-# operate (via CLDR's ant/CLDRConverterTool.java and ant/CLDRBuild.java) the
-# necessary CLDR tools including LDML2ICUConverter, ConvertTransforms, etc.
-# This process will take several minutes.
-# Keep a log so you can investigate anything that looks suspicious.
+# 4a. Generate the CLDR production data, and clean out the existing ICU4C
+# data in preparation for the newly generated data (this is important when
+# the new data may  no longer generate a data file that is currently present).
+# This process uses ant with ICU's data/build.xml
+#
+# Running "ant cleanprod" is necessary to clean out the production data directory
+# (usually $CLDR_TMP_DIR/production ), required if any CLDR data has changed.
 #
 # Running "ant setup" is not required, but it will print useful errors to
 # debug issues with your path when it fails.
 #
+# Running "ant clean" cleans out the existing ICU4C data.
 
 cd $ICU4C_DIR/source/data
+ant cleanprod
 ant setup
 ant clean
-ant all 2>&1 | tee /tmp/cldr-newData-buildLog.txt
+ant proddata 2>&1 | tee /tmp/cldr-newData-proddataLog.txt
 
-# NOTE: if you change the CLDR data, please run "ant cleanprod" to clean out the
-# temporary production data directory (usually $CLDR_DIR/../cldr-aux/production )
+# 4b. Build the new ICU4C data files; these include .txt files and .py files.
+# These new files will replace whatever was already present in the ICU4C sources.
+# This process uses the LdmlConverter in $TOOLS_ROOT/cldr/cldr-to-icu/
+#
+# This process will take several minutes,, during most of which there will be no log
+# output (so do not assume nothing is happening). Keep a log so you can investigate
+# anything that looks suspicious.
+#
+# This also requires temporarily redefining CLDR_DIR.
+
```

Comment:
We should definitely fix this to not need this sort of messing about.

Reply:
OK, thanks, yes I could do that. I don't see a need to immediately update the doc, I will fix it the next time I do a CLDR integration (it will happen at least 2 more times before we are done with ICU 67).